### PR TITLE
Add Contributor Covenant as Code of Conduct

### LINKED ISSUE #123: Add code of conduct
A code of conduct is very useful for a project from the beginning, even if it's not so complicated. A very popular one that works for most software projects is the [Contributor Covenant](https://www.contributor-covenant.org/).

This document can be added in several places (see https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project).

### PR DESCRIPTION
**Pull request by cthoyt**
Closes #123

This PR adopts the [Contributor Covenant](https://www.contributor-covenant.org) as a simple yet effective code of conduct for this project. It names @cmungall as the contact person for issue resolution. This PR also adds the related badge to the README and mentions that contributors are responsible for upholding this COC

### CODE REVIEW

**Review by cmungall — APPROVED** [review]

**Review by matentzn — APPROVED** [review]
Using the same also for new ontology repos

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Python lib for common ontology operations over a variety of backends.
 [![badge](https://img.shields.io/badge/launch-binder-579ACA.svg)](https://mybinder.org/v2/gh/incatools/ontology-access-kit/main?filepath=notebooks)
 [![Downloads](https://pepy.tech/badge/oaklib/week)](https://pepy.tech/project/oaklib)
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.6456239.svg)](https://doi.org/10.5281/zenodo.6456239)
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](.github/CODE_OF_CONDUCT.md)
```
Comment (nlharris) [inline README.md:10]: Later, this document is referred to as the "Code of Conduct". Would it make sense to use that terminology here as well? you could put both:
Contributor Covenant (Code of Conduct)

**Review by nlharris — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (cthoyt) [reply README.md:10]: contributor covenant is the name of the document, similar to how "MIT License" is the title on many repository's "LICENSE" file

File: .github/CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+cjm@berkeleybop.org (Christopher J Mungall).
```
Comment (sierra-moxon) [inline .github/CODE_OF_CONDUCT.md:63]: Is this the email address we should use @cmungall ?

**Review by sierra-moxon — APPROVED** [review]

File: .github/CODE_OF_CONDUCT.md (hunk 2 above)
Comment (cthoyt) [reply .github/CODE_OF_CONDUCT.md:63]: that's what's written in the package's installation metadata

Comment (cthoyt) [reply .github/CODE_OF_CONDUCT.md:63]: https://github.com/cthoyt/ontology-access-kit/blob/f83966d38480c461a50d7980cde48eb8d800bf3c/pyproject.toml#L1-L5

Comment (nlharris) [reply .github/CODE_OF_CONDUCT.md:63]: I wasn't suggesting that you made it up! I will check with Chris about what he wants to do.

Comment (cmungall) [reply .github/CODE_OF_CONDUCT.md:63]: use my lbl. My github is associated with my bbop alias so often will github will auto-surface it in various autogen docs. uberon uses my lbl email. yes, i should simplify my life

**Pull request merged**